app/components/bali/card/index.scss

```diff
@@ -0,0 +1,18 @@
+.column:not(.is-full) {
```

Comment:
todo el CSS de un componente debe estar scoped bajo una clase con el nombre del componente. Ej. `card-component`.

Esto para que el CSS de cada componente solo impacte a los elementos del componente y no tenga impacto en otros elementos.

Reply:
Está bien si lo dejo como .card-component.card desde la línea 2? Tengo que evaluar si viene de una columna superior y no es "is-full", para que no le de estas alturas fijas al leader post